script/local

```diff
@@ -51,4 +50,4 @@ RAILS_COMMANDS = %w{generate console server dbconsole g c s runner}
 args = RAILS_COMMANDS.include?(ARGV[0]) ? ["rails", ARGV].flatten : ARGV
 
 # Run the command
-exec "cd test-rails-app && GEMFILE=../Gemfile #{args.join(" ")}"
+exec "cd test-rails-app && #{args.join(" ")}"
```

Comment:
Yes, I should probably squash https://github.com/activeadmin/activeadmin/pull/4763/commits/f495ef6d505fbaa876fbf1be8a293b6a39a8510f and https://github.com/activeadmin/activeadmin/pull/4763/commits/ad6707de6e93e52b1dabde2d6aedad19c8e4d7f1 since they are part of the same thing. There's plenty of reasons to do this:Previously I couldn't create the sample application because of the error in https://github.com/activeadmin/activeadmin/pull/4763/commits/ad6707de6e93e52b1dabde2d6aedad19c8e4d7f1 which is pretty self descriptive. I don't know what that `GEMFILE` environment variable does, I guess it makes the `rails` command pick up that `Gemfile`but it's definitely not the best way to go. Why create the sample application without a `Gemfile` and then point to an external `Gemfile` to make it work?